.circleci/docker/common/install_cudnn8.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+# cuDNN license: https://developer.nvidia.com/cudnn/license_agreement
+mkdir tmp_cudnn && cd tmp_cudnn
+wget -q https://developer.download.nvidia.com/compute/redist/cudnn/v8.3.2/local_installers/11.5/cudnn-linux-x86_64-8.3.2.44_cuda11.5-archive.tar.xz -O cudnn-linux-x86_64-8.3.2.44_cuda11.5-archive.tar.xz
+tar xf cudnn-linux-x86_64-8.3.2.44_cuda11.5-archive.tar.xz
+cp -a cudnn-linux-x86_64-8.3.2.44_cuda11.5-archive/include/* /usr/local/cuda/include/
+cp -a cudnn-linux-x86_64-8.3.2.44_cuda11.5-archive/lib/* /usr/local/cuda/lib64/
```

Comment:
Please try to avoid string magic duplication, use CUDNN_URL as base and infer the rest of names from it
```suggestion
CUDNN_URL="https://developer.download.nvidia.com/compute/redist/cudnn/v8.3.2/local_installers/11.5/cudnn-linux-x86_64-8.3.2.44_cuda11.5-archive.tar.xz "
CUDNN_NAME=$(basename ${CUDNN_URL})
CUDNN_FOLDER=${CUDNN_NAME:/.}
wget -q ${CUDNN_URL} -O ${CUDNN_NAME}
tar xf ${CUDNN_NAME}
cp -a ${CUDNN_FOLDER}/include/* /usr/local/cuda/include/
cp -a ${CUDNN_FOLDER}/lib/* /usr/local/cuda/lib64/
```